layers/amd64_wsl2/etc/containers/containers.conf

```diff
@@ -0,0 +1,13 @@
+[containers]
+cgroups = "disabled"
```

Comment:
Due to the lack of [CDI device description](https://github.com/cncf-tags/container-device-interface/blob/main/SPEC.md), we currently disable "Cgroups" so that "nvidia-sim" and other programs that require nvidia drivers can call the Nvidia driver stack without crashing